packages/jest-haste-map/src/crawlers/watchman.js

```diff
@@ -159,12 +160,15 @@ module.exports = async function watchmanCrawl(
 
   for (const [watchRoot, response] of watchmanFiles) {
     const fsRoot = normalizePathSep(watchRoot);
-    clocks.set(fsRoot, response.clock);
+    const relativeFsRoot = path.relative(rootDir, fsRoot);
```

Comment:
You are adding two path.relative calls here. How does this affect performance of the initial crawl?

Reply:
I'm still running some tests, but it basically cancels the benefits of the `Map` change. I'm trying to optimize it at least for the most common cases.

Reply:
I managed to make the calls to resolve and relative negligible. See `fast_path.js`.